WS2012R2/lisa/Infrastructure/Analyze-IperfResultsThroughputFromLinuxSar.ps1

```diff
@@ -0,0 +1,232 @@
+﻿param([string] $vmName, [string] $hvServer, [string] $testParams)
+
+#
+# Check input arguments
+#
+if ($vmName -eq $null)
```

Comment:
Parameter checking in PowerShell is best done via [ValidateScript] attribute. See below link.

http://blogs.technet.com/b/heyscriptingguy/archive/2011/05/15/simplify-your-powershell-script-with-parameter-validation.aspx